google/resource-snippets/tpu-v1alpha1/tpu.yaml

```diff
@@ -5,12 +5,13 @@ resources:
 - name: tpu
   type: tpu.py
   properties:
-    zone: ZONE_TO_RUN
```

Comment:
we use this placeholder in our test suite to configure the cloud zone to use, was it causing any issues?